hazelcast/src/main/java/com/hazelcast/wan/impl/WanReplicationServiceImpl.java

```diff
@@ -163,4 +164,9 @@ public void shutdown() {
     private ConcurrentHashMap<String, WanReplicationPublisherDelegate> initializeWanReplicationPublisherMapping() {
         return new ConcurrentHashMap<String, WanReplicationPublisherDelegate>(2);
     }
+
+    @Override
+    public Map<String, LocalWanStats> getStats() {
```

Comment:
How can it be that an interface needs to be implemented, but the actual implementation returns null?

Is additional work required? If so, please add a comment.

And returning null if there is nothing to do is often a PITA since it forces the caller to do null checks. Often it is best to return an empty map (see EMPTY_MAP from Collections). This way you get less branching on the calling side.